f5lbaasdriver/v2/bigip/neutron_client.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
```

Comment:
Should the setup.py for the driver ensure the neutron client is there and is at the right version?